scripts/utils/trading_strategy_helpers.js

```diff
@@ -70,6 +70,38 @@ const maxUINT = new BN(2).pow(new BN(256)).sub(new BN(1))
  * @property {Address} tokenAddresses List of tokens that the traded wishes to withdraw
  */
 
+/**
+ * @typedef TokenObject
+ *  * Example:
+ * {
+ *   symbol: "WETH",
+ *   decimals: 18,
+ *   tokenAddress: "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
+ * }
+ * @type {object}
+ * @property {string} symbol symbol representing the token
+ * @property {(number|BN)} decimals number of decimals of the token
+ * @property {address} [tokenAddress] address of the token contract on the EVM
+
+ */
+
+/**
+ * Returns an instance of the exchange contract
+ */
+const getExchange = async function(web3) {
```

Comment:
does this function needs to be async, if we don't await the promise, just return the promise